examples/VASP/water-1-npt-md-vasp/POTCAR

```diff
@@ -0,0 +1,4049 @@
+  PAW_PBE O 08Apr2002                    
+   6.00000000000000     
+ parameters from PSCTR are:
+   VRHFIN =O: s2p4
```

Comment:
I have no idea about VASP calcs but this file is enourmous :) isn't it a standard file? it needs to be here?

Reply:
Dear Mariana, 

Sorry for the late reply, I should perhaps check my github account more often. Regarding your comment, this is created from standard files, you need to cat the pesudo-potentiuals for each element in the system (in the same order as specified in POSCAR), and name it POTCAR. This POTCAR file must be pasted into the running directory for the calculation to run. I wonder what's the best solution for this case?

Wei